frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -39,16 +41,22 @@ const PAGE_LIMIT = 25;
 
 const PaginatedProjectOverview = () => {
     const projectId = useRequiredPathParam('projectId');
+    const [searchParams, setSearchParams] = useSearchParams();
     const { project, loading: projectLoading } = useProject(projectId, {
         refreshInterval,
     });
     const [currentOffset, setCurrentOffset] = useState(0);
+
+    const [searchValue, setSearchValue] = useState(
+        searchParams.get('search') || '',
+    );
+
     const {
         features: searchFeatures,
         total,
         refetch,
         loading,
-    } = useFeatureSearch(currentOffset, PAGE_LIMIT, projectId, {
+    } = useFeatureSearch(currentOffset, PAGE_LIMIT, projectId, searchValue, {
```

Comment:
Are we using this data for something other than passing it into the project feature toggles component? If not, maybe we can move it inside.

Reply:
We are using the data for paging.